src/olympia/ratings/tests/test_views.py

```diff
@@ -2400,16 +2400,30 @@ def test_replying_to_a_reply_is_not_possible(self):
     def test_throttle(self):
         self.addon_author = user_factory()
         self.addon.addonuser_set.create(user=self.addon_author)
-        self.client.login_api(self.addon_author)
+        other_rating = Rating.objects.create(
+            addon=self.addon, version=self.addon.current_version, rating=1,
+            body='My review', user=user_factory())
+        other_url = reverse(self.reply_url_name,
+                            kwargs={'pk': other_rating.pk})
 
-        # First post, no problem.
-        response = self.client.post(self.url, data={
-            'body': u'My réply...',
-        })
-        assert response.status_code == 201
+        with freeze_time('2017-11-01') as frozen_time:
+            self.client.login_api(self.addon_author)
+            # First post, no problem.
+            response = self.client.post(self.url, data={
+                'body': u'My réply...',
+            })
+            assert response.status_code == 201
 
-        # Second post, nope, have to wait a while.
-        response = self.client.post(self.url, {
-            'body': u'Another réply',
-        })
-        assert response.status_code == 429
+            # Second post, nope, have to wait a while.
```

Comment:
Can you force a 4 seconds wait just to be sure?

Reply:
https://github.com/mozilla/addons-server/pull/8376/commits/fe7296e453d396f2751d0a6fa2dc23c1750afc5f